pyproject.toml

```diff
@@ -0,0 +1,117 @@
+[build-system]
+
+requires = [
+    "setuptools>=45.0",
+    "setuptools_scm[toml]>=6.2",
+    "wheel>=0.38.1",
```

Comment:
I don't think you need to include wheel

Reply:
You are right. I have excluded `wheel` now